deploy/install.sh

```diff
@@ -2,367 +2,14 @@
-
-  if echo "$response" | grep -Fq "${CONNECTION_NAME}"; then
-    echo "Kubernetes monitoring successfully setup."
-  else
-    echo "Error adding Kubernetes cluster to Dynatrace: $response"
-  fi
-}
-
-checkForExistingCluster() {
-  response=$(apiRequest "GET" "/config/v1/kubernetes/credentials" "")
-
-  # To check the exact name we need to include the `\` at the end
-  if echo "$response" | grep -Fq "\"name\":\"${CONNECTION_NAME}\""; then
-    echo "Error: Cluster already exists: ${CONNECTION_NAME}"
-    exit 1
-  fi
-  # To check the endpoint URL we must not include the `\` at the end
-  if echo "$response" | grep -Fq "\"endpointUrl\":\"${K8S_ENDPOINT}"; then
-    echo "Error: Cluster endpoint already exists: ${K8S_ENDPOINT}"
-    exit 1
-  fi
-}
-
-checkTokenScopes() {
-  jsonAPI="{\"token\": \"${API_TOKEN}\"}"
-  jsonPaaS="{\"token\": \"${PAAS_TOKEN}\"}"
-
-  responseAPI=$(apiRequest "POST" "/v1/tokens/lookup" "${jsonAPI}")
-
-  if echo "$responseAPI" | grep -Fq "Authentication failed"; then
-    echo "Error: API token authentication failed!"
-    exit 1
-  fi
-
-  if ! echo "$responseAPI" | grep -Fq "WriteConfig"; then
-    echo "Error: API token does not have config write permission!"
-    exit 1
-  fi
-
-  if ! echo "$responseAPI" | grep -Fq "ReadConfig"; then
-    echo "Error: API token does not have config read permission!"
-    exit 1
-  fi
-
-  if echo "$responseAPI" | grep -Fq '"revoked": true'; then
-    echo "Error: API token has been revoked!"
-    exit 1
-  fi
-
-  responsePaaS=$(apiRequest "POST" "/v1/tokens/lookup" "${jsonPaaS}")
-
-  if ! $USES_API_AS_PAAS_TOKEN; then
-    if echo "$responsePaaS" | grep -Fq "Token does not exist"; then
-      echo "Error: PaaS token does not exist!"
-      exit 1
-    fi
-
-    if echo "$responsePaaS" | grep -Fq '"revoked": true'; then
-      echo "Error: PaaS token has been revoked!"
-      exit 1
-    fi
-  fi
-}
-
-apiRequest() {
-  method=$1
-  url=$2
-  json=$3
-
-  if "$SKIP_CERT_CHECK" = "true"; then
-    curl_command="curl -k"
-  else
-    curl_command="curl"
-  fi
-
-  response="$(${curl_command} -sS -X ${method} "${API_URL}${url}" \
-    -H "accept: application/json; charset=utf-8" \
-    -H "Authorization: Api-Token ${API_TOKEN}" \
-    -H "Content-Type: application/json; charset=utf-8" \
-    -d "${json}")"
-
-  echo "$response"
-}
-
-####### MAIN #######
-printf "\nCheck for token scopes...\n"
-checkTokenScopes
-printf "\nCheck if cluster already exists...\n"
-checkForExistingCluster
-printf "\nCreating Dynatrace namespace...\n"
-checkIfNSExists
-printf "\nApplying Dynatrace Operator...\n"
-applyDynatraceOperator
-printf "\nWait for webhook to become available\n"
-waitForWebhook
-printf "\nApplying DynaKube CustomResource...\n"
-applyDynaKubeCR
-printf "\nAdding cluster to Dynatrace...\n"
-addK8sConfiguration
+bold=$(tput bold)
+normal=$(tput sgr0)
+
+echo ${bold}This script is deprecated - installation aborted.${normal}
```

Comment:
I don't think we should format anything in bold in the output. Just the normal text is ok.

Reply:
Done